src/biome/text/modules/heads/__init__.py

```diff
@@ -1,11 +1,11 @@
 # fmt: off
-from .bimpm_classification import BiMpm
 from .defs import TaskHead, TaskHeadSpec, TaskName, TaskOutput
 from .doc_classification import DocumentClassification
 from .language_modelling import LanguageModelling, LanguageModellingSpec
 from .record_classification import RecordClassification, RecordClassificationSpec
 from .text_classification import TextClassification, TextClassificationSpec
 from .token_classification import TokenClassification, TokenClassificationSpec
+from .record_bimpm_classification import RecordBiMpm
```

Comment:
As you mentioned `RecordPairClassification` is a good name for this.

Reply:
Done